src/job.ts

```diff
@@ -829,14 +829,22 @@ export class Job {
     }
 
     private async pullImage (writeStreams: WriteStreams, imageToPull: string) {
-        try {
-            await Utils.spawn([this.argv.containerExecutable, "image", "inspect", imageToPull]);
-        } catch (e: any) {
+        const pullPolicy = this.argv.pullPolicy;
+        const actualPull = async () => {
             const time = process.hrtime();
             await Utils.spawn([this.argv.containerExecutable, "pull", imageToPull]);
             const endTime = process.hrtime(time);
             writeStreams.stdout(chalk`${this.formattedJobName} {magentaBright pulled} ${imageToPull} in {magenta ${prettyHrtime(endTime)}}\n`);
             this.refreshLongRunningSilentTimeout(writeStreams);
+        };
+
+        if (pullPolicy === "always") {
+            return actualPull();
```

Comment:
```suggestion
          await actualPull();
          return 
```

Reply:
Whats the different?

Reply:
My initial motivation was to be more consistent, 

since for the flow when `pullPolicy !== "always:`, 

```javascript
        try {
            await Utils.spawn([this.argv.containerExecutable, "image", "inspect", imageToPull]);
        } catch (e: any) {
            await actualPull();
        }
 ```

Reply:
@ANGkeith I've applied it.

Reply:
was doing some readings as well, seemed like there's some subtle difference with with the stacktrace. 

`return promise` vs `return await promise`

https://github.com/goldbergyoni/nodebestpractices/blob/master/sections/errorhandling/returningpromises.md#code-example-anti-pattern-calling-async-function-without-awaiting